packages/solecs/removeBytecode.ts

```diff
@@ -0,0 +1,17 @@
+import fs from "fs";
+
+export default function removeBytecode(abiDir: string) {
+  for (const abiFile of fs.readdirSync(abiDir)) {
+    if (!abiFile.match(/.*\.json$/)) continue;
+    const realpath = fs.realpathSync(`${abiDir}/${abiFile}`);
+
+    const abi = JSON.parse(fs.readFileSync(realpath).toString("utf8"));
+    delete abi["bytecode"];
+    delete abi["deployedBytecode"];
+    delete abi["ast"];
+
+    fs.writeFileSync(realpath, JSON.stringify(abi, null, 2) + "\n");
+  }
+}
+
+removeBytecode("abi/");
```

Comment:
@alvrs @ludns Is there a package to put shared utils like this so we don't have to duplicate code around? If not, should there be?

Reply:
yes `utils` in this repo
`@latticexyz/utils`

Reply:
I had the same thought, but noticed utils wasn't already imported and decided against it. Can move it there then in a bit